### DISCUSSION TITLE
Add Find Element Index concept

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#45] [Q&A] [answered]
The new Find content option is great for finding specific entries.

With the whole entrification concept, our section lists are getting pretty large. Hopefully Craft solves this at some point, but in the meantime I use a custom url function to create quick links to element indexes in the CP.

```
$sections = BusinessLogicModule::getInstance()
            ->helpers->getSections()
            ->map(
                fn($e) => [
                    "name" => "Index " . $e->name,
                    "url" => UrlHelper::cpUrl("entries/" . $e->handle),
                    "subtitle" => "Element Index > " . $e->name,
                ],
            );

        return $sections->toArray();
```

Something like this (where that businesslogic function is just returning the sections of the site).

Could definitely see something like that being built in though.

I guess related to [this](https://github.com/trendyminds/craft-palette/discussions/44), it seems custom urls are weighted less than the built in functions.

So if I have a section named "Pages", the items for editing the section settings or entry type settings show first instead of my custom url. So a shortcut to jump to that would be handy, or if I could weight custom urls over the built in ones.

I guess there's 2 ideas here...
1. If quick linking to various element indexes should be a built in function
2. If custom urls could/should weight higher than built in items

Thanks! As I've said before, as someone who uses shortcuts for everything, this plugin is :chefkiss:

**Comment by aaronbushnell** [comment, marked as answer]
Hey @kylecotter, thanks so much for the kind words! ☺️ 

I've just pushed out 5.1.0 and 4.1.0 that now exposes section-specific links within the "Entries" section. You should be able to easily jump to, for example, "Blog Entries" and "Staff Entries" natively.

Hope that helps with your use-case here!
